pkg/apis/pipeline/v1alpha1/task_types.go

```diff
@@ -59,6 +59,11 @@ type TaskSpec struct {
 var _ apis.Validatable = (*Task)(nil)
 var _ apis.Defaultable = (*Task)(nil)
 
+const (
+	// TaskOutputImageDefaultDir is the default directory for output image resource,
+	TaskOutputImageDefaultDir = "/builder/home/image-outputs"
```

Comment:
we seem to be hardcoding the `/builder/home` path a bunch in https://github.com/tektoncd/pipeline/blob/3d87e2e18174f1d7b088bdd14065d5f06ef86462/pkg/reconciler/v1alpha1/taskrun/resources/pod.go - instead of hardcoding it in these 2 places, maybe we could define it once and share it?

Reply:
ok